packages/contact-center/cc-widgets/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@webex/cc-widgets",
+  "description": "Webex Contact Center Widgets",
+  "version": "1.0.0",
```

Comment:
This will get updated with the latest version by semantic-release once we merge the PR.